skits/pipeline.py

```diff
@@ -267,16 +271,16 @@ def forecast(self, X, start_idx, trans_window=None):
 
         # Create the final prediction matrix and fill in all predictions up to
         # the starting index.
-        X_total = np.empty((X.shape[0], 1), dtype=np.float32)
+        X_total = np.empty((end_idx, 1), dtype=np.float32)
         X_total[:X_init.shape[0], 0] = X_init
 
         # For each out of sample point (aka >= start_idx)
-        for idx in range(start_idx, len(X)):
+        for idx in range(start_idx, end_idx):
 
             # Predict the next point
-            offset = trans_window or idx
+            offset = trans_window or np.inf
```

Comment:
The use of `np.inf` and `max` below is a little confusing. Is there a way to make this more explicit?

Reply:
For sure, it was convenient haha - I made these changes quickly because I'm actively using skits for forecasting. Great library by the way.